devices/heiman/HS2IRC_IR_control.json

```diff
@@ -0,0 +1,72 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "HEIMAN",
+  "modelid": "IRControl-EM",
+  "vendor": "HEIMAN",
+  "product": "HS2IRC",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$ZHAControl",
```

Comment:
@Smanar I think it's `$TYPE_CONTROL_SENSOR` here

Reply:
Ofc ^^.
Corrected